### PR TITLE
Fix int overflow

### PR DESCRIPTION
**Pull request by cpuguy83**
Closes #27

### CODE REVIEW

**Comment by cpuguy83** [comment]
Test run here: https://github.com/cpuguy83/go-ansiterm/pull/5/checks?check_run_id=2853454830

Apparently there's another bug as well which should be fixed by #27

**Comment by cpuguy83** [comment]
@jstarks @kevpar

File: winterm/ansi.go [hunk 1]
```diff
@@ -163,15 +163,28 @@ func ensureInRange(n int16, min int16, max int16) int16 {
 
 func GetStdFile(nFile int) (*os.File, uintptr) {
 	var file *os.File
-	switch nFile {
-	case syscall.STD_INPUT_HANDLE, windows.STD_INPUT_HANDLE:
+
+	// syscall uses negative numbers
+	// windows package uses very big uint32
+	// Keep these switches split so we don't have to convert ints too much.
+	switch uint32(nFile) {
```
Comment (thaJeztah) [inline winterm/ansi.go:170]: I went over a few variations as well; would casting to `int64` work? https://github.com/docker/cli/pull/3131/commits/97da45d9a187ae1d697ed2023bd05b0fd6c47602

Comment (cpuguy83) [reply winterm/ansi.go:170]: It does, but I figured it would be better to split since these are really different types.

Comment (thaJeztah) [reply winterm/ansi.go:170]: Yeah, I think ultimately we should drop the syscall one (and introduce a breaking change once)

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by jstarks** [comment]
This time we got it for sure.

**Pull request merged**